.changeset/chatty-socks-occur.md

```diff
@@ -0,0 +1,13 @@
+---
+'@leafygreen-ui/number-input': major
+---
+
+[LG-4134](https://jira.mongodb.org/browse/LG-4134)
+
+1. Updated styling:
```

Comment:
Not for now, but we should write some guidelines for changesets. (Tense consistency, when to use ols/uls, etc.) 

Maybe you have a POV here

Reply:
I'm sure there's a good one written somewhere. I'll take a look around and share with the team at the next dev sync. I've mostly been writing based on how content reads/appears in the generated changelog files